checker/requestcache.go

```diff
@@ -100,6 +125,20 @@ func (r *requestCache) initPeer(aPeer *authz.AttributeContext_Peer) (*peer, erro
 	return &peer, nil
 }
 
+func (r *requestCache) initNamespace(name string) *namespace {
+	ns := &namespace{Name: name}
+	// If the namespace is in the store, copy labels over.
+	id := proto.NamespaceID{Name: name}
+	msg, ok := r.store.NamespaceByID[id]
```

Comment:
should we use the `store` read api. May be safer?

Reply:
This gets called from inside a `read` block in `Check` (`server.go`)

We don't want to call `read` again since that would create a re-entrant lock.

Reply:
ahh.. makes sense!